DoSomethingArchive/legacy-website
# Forge 6.6

# Changes

Updates for [Forge 6.6](https://github.com/DoSomething/forge/releases/tag/v6.6.0). The biggest changes are just removing code that was moved into the latest Forge release. I also removed two mix-ins that were only being used once each, since they seemed to just be adding more complexity.

For review: @DoSomething/front-end 
# How should I test this?

Ideally, pull down this branch & run `npm install && grunt build` from the theme directory. The hot/win module & reportback confirmation page should look roughly identical (and _certainly_ not broken).
# Screenshots

![screen shot 2015-09-10 at 3 13 31 pm](https://cloud.githubusercontent.com/assets/583202/9798411/c23271c8-57ce-11e5-8d41-cd4bf5a34088.png)

![screen shot 2015-09-10 at 3 14 08 pm](https://cloud.githubusercontent.com/assets/583202/9798421/c7404c4e-57ce-11e5-8a8c-8e4474b925f2.png)

![screen shot 2015-09-10 at 3 15 14 pm](https://cloud.githubusercontent.com/assets/583202/9798422/ca2670d2-57ce-11e5-8e07-479a70df34fd.png)
